packages/core/src/Stores/notification-store.js

```diff
@@ -36,7 +36,7 @@ export default class NotificationStore extends BaseStore {
     marked_notifications = [];
     push_notifications = [];
     client_notifications = {};
-    should_show_popups = true;
+    should_show_popups = false;
```

Comment:
@yauheni-kryzhyk-deriv since we no longer use/set `should_show_popups`, this variable should be completely removed together with `setShouldShowPopups` 🙂

Reply:
Yes, I agree, we could also delete <NotificationsContent/> as i isn’t invoked.
but I decides to save this functionality for further development: we have other card for improvement the notifications behaviour. So I believe we will need all this stuff.